javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,74 @@
+//Find the shortest world
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function findShortestWord(array) {
+  let currentWord;
```

Comment:
You can assign "danishWords[0]" to "currentWord" here. Instead of inside the for loop.